deploy.sh

```diff
@@ -20,21 +21,28 @@ while getopts "pha:-:" OPT; do
   fi
   case "$OPT" in
     h | help)
-      echo "Builds and deploys the contract with state initialization (first deployement)."
+      echo "Builds and deploys contracts, eventually with state initialization."
```

Comment:
"eventually" seems like an unnecessary word here.

```suggestion
      echo "Builds and deploys contracts with state initialization."
```